src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -121,6 +124,12 @@ public Command parseCommand(String userInput) throws ParseException {
             case SORT:
                 return new SortEventCommandParser().parse(arguments);
 
+            case REMIND:
+                return new RemindEventCommandParser().parse(arguments);
+
+            case REMINDER:
+                return new ListReminderEventCommand();
+
```

Comment:
Theses cases are ambigously named when they clearly mean different things. And is separated from the show reminder command when they should be grouped under another command type for reminder. Instead of belonging to events.